application/common/models/Invite.php

```diff
@@ -138,4 +138,31 @@ public function renew(): string
 
         return $this->getCode();
     }
+
+    /**
+     * Delete all invite records where expires_on date is in the past
+     * by more than a configured "grace period."
+     */
+    public static function deleteOldInvites()
+    {
+        /*
+         * Replace '+' with '-' so all env parameters can be defined consistently as '+n unit'
+         */
+        $inviteGracePeriod = str_replace('+', '-', \Yii::$app->params['inviteGracePeriod']);
+        $removeOlderThan = MySqlDateTime::relative($inviteGracePeriod);
```

Comment:
The logic is correct, but maybe the variable name `removeOlderThan` is misleading. I changed it to `removeExpireBefore`. Since the expire field in the database already factors in the lifetime (expiration time) we just need to subtract the grace period from the current date/time, and compare the result against the expire field.